src/styles/GlobalStyles.ts

```diff
@@ -31,6 +31,10 @@ export const GlobalStyles = createGlobalStyle`
     font-weight: 600;
     src: url('fonts/Mont-Regular.otf') format('opentype');
   }
+  
```

Comment:
Don't use * selector in css. Always use the tag that you want some property. * can lead performace issues.